docker-multinode/common.sh

```diff
@@ -36,18 +39,35 @@ kube::multinode::main(){
   NET_INTERFACE=${NET_INTERFACE:-${DEFAULT_NET_INTERFACE}}
 
   TIMEOUT_FOR_SERVICES=${TIMEOUT_FOR_SERVICES:-20}
+  USE_CNI=${USE_CNI:-"false"}
+  CNI_ARGS=""
 
   # Constants
   BOOTSTRAP_DOCKER_SOCK="unix:///var/run/docker-bootstrap.sock"
+  BOOTSTRAP_DOCKER_PARAM="-H ${BOOTSTRAP_DOCKER_SOCK}"
   KUBELET_MOUNTS="\
     -v /sys:/sys:rw \
     -v /var/run:/var/run:rw \
+    -v /run:/run:rw \
     -v /var/lib/docker:/var/lib/docker:rw \
     -v /var/lib/kubelet:/var/lib/kubelet:shared \
     -v /var/log/containers:/var/log/containers:rw"
 
   # Paths
-  FLANNEL_SUBNET_TMPDIR=$(mktemp -d)
+  FLANNEL_SUBNET_DIR=${FLANNEL_SUBNET_DIR:-/run/flannel}
+
+  kube::helpers::parse_version ${K8S_VERSION}
+
+  if [[ ${USE_CNI} == "true" && \
```

Comment:
Yes, I'll add that later.
CNI should be automatically chosen with version `v1.4.0-alpha.2` and above when we consider it stable.